doc/tutorials/custom-builds.md

```diff
@@ -223,6 +223,10 @@ The exports are given here in the order in which they occur in the `heatmaps-ear
 
 As this example uses a vector layer it is necessary to remove `"ol.ENABLE_VECTOR=false"` in the `define` section of the configuration.
 
+## Maintaining the code
+
+If you installed OpenLayers from the Node package, you can use `npm` to upgrade to the latest version. If you cloned the Github repo, simply pulling in the latest code may not be enough, as some of the packages used, for example, the compiler, may need upgrading too. Do this by using `npm install` rather than `npm update`.
```

Comment:
This document only talks about installing OpenLayers from the npm package, so the information about the cloned repo is misleading. All that'd be needed to get the latest would be to run `npm install openlayers`.